pwem/protocols/protocol_sets.py

```diff
@@ -139,13 +140,19 @@ def createOutputStep(self):
 
         # Read ClassName and create the corresponding EMSet (SetOfParticles...)
         try:
-            outputSetFunction = getattr(self, "_create%s" % set1.getClassName())
+            if str(set1.getClassName()) is not 'Volume':
```

Comment:
I now 'Volume' is in many places, but we should try to improve this having a single class name source.

I'd suggest to replace 'Volume' by  `Volume.__name__`

Reply:
Done